.github/workflows/manifests_example_test.yaml

```diff
@@ -19,4 +19,4 @@ jobs:
 
     - name: Unit Test
       run: |
-        kustomize build example
+        kustomize build --enable-helm example
```

Comment:
We should not change the default installation for now due to politcial reasons. Lets do the spark -operator stuff under experimental/helm for now. and keep 
apps/spark/spark-operator as it is

Reply:
We can leave the PR open, just create a separate one